_data/modules/thirdparty.yaml

```diff
@@ -10,6 +10,15 @@
 ## repository to update compatibility info.
 
 ## Bundles that have been tested on Dropwizard 1.3
+- name: armeria-dropwizard
+  url: https://github.com/line/armeria/dropwizard
```

Comment:
I think this will become https://github.com/line/armeria/tree/master/dropwizard once https://github.com/line/armeria/pull/2236 is merged in

Reply:
Whoops, yeah.